draft-ietf-lisp-pubsub.xml

```diff
@@ -284,19 +285,19 @@ Rec +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 
-          <t>If the subscription request fails, the Map-Server MUST send a Map-Reply to the originator of the Map-Request, as described in Section 8.3 of <xref target="RFC9301"></xref>. The xTR processes the Map-Reply as specified in Section 8.1 of <xref target="RFC9301"></xref>.</t>
+          <t>If the subscription request fails, the Map-Server MUST send a Map-Reply to the originator of the Map-Request, as described in Section 8.3 of <xref target="RFC9301"></xref>. The xTR processes the Map-Reply as specified in Section 8.1 of <xref target="RFC9301"></xref>. If the subscription request fails, it is up to the implementation to try to subscribe again. </t>
 
           <t>If the Map-Server receives a subscription request for an EID-Prefix not present in the mapping database, it SHOULD follow the same logic described in Section 8.4 of <xref target="RFC9301"></xref> and create a temporary subscription state for the xTR-ID to the least-specific prefix that both matches the original query and does not match any EID-Prefix known to exist in the LISP-capable infrastructure. Alternatively, the Map-Server can instead determine that such subscription request fails, and send a Negative Map-Reply following Section 8.3 of <xref target="RFC9301"></xref>. In both cases, the TTL of the temporary subscription state or the Negative Map-Reply SHOULD be configurable, with a value of 15-minutes being RECOMMENDED. </t>
 
-          <t>The subscription state can also be created explicitly by configuration at the Map-Server (possible when a pre-shared security association exists, see <xref target="security"></xref>). In this case, the initial nonce associated with the xTR-ID (and EID-Record) MUST be randomly generated by the Map-Server.  </t>
+          <t>The subscription state can also be created explicitly by configuration at the Map-Server (possible when a pre-shared security association exists, see <xref target="security"></xref>). In this case, the initial nonce associated with the xTR-ID (and EID-Prefix) MUST be randomly generated by the Map-Server.</t>
 
           <t>The following specifies the procedure to remove a subscription: If the Map-Request only has one ITR-RLOC with AFI = 0 (i.e., Unknown Address), the Map-Server MUST remove the subscription state for that xTR-ID. In this case, the Map-Server MUST send the Map-Notify to the source RLOC of the Map-Request. If the Map-Server has received this Map-Request for an EID-Prefix without explicit subscription state for that xTR-ID, but covered by a less-specific EID-Prefix for which subscription state exists for the xTR-ID, the Map-Server SHOULD stop publishing updates about this more-specific EID-Prefix to that xTR, until the xTR explicitly subscribes to the more-specific EID-Prefix.</t>
```

Comment:
```suggestion
          <t>The following specifies the procedure to remove a subscription: If the Map-Request only has one ITR-RLOC with AFI = 0 (i.e., Unknown Address) and a nonce that is greater than the stored by the Map-Server, the Map-Server MUST remove the subscription state for that xTR-ID. In this case, the Map-Server MUST send the Map-Notify to the source RLOC of the Map-Request. If the Map-Server has received this Map-Request for an EID-Prefix without explicit subscription state for that xTR-ID, but covered by a less-specific EID-Prefix for which subscription state exists for the xTR-ID, the Map-Server SHOULD stop publishing updates about this more-specific EID-Prefix to that xTR, until the xTR explicitly subscribes to the more-specific EID-Prefix. Similar to updating existing subscriptions, Map-Requests used for withdrawal MUST be silently ignored of the nonce check fails.</t>
```

Reply:
I worry we are defining the same behavior twice. I would rather have a sentence here that just says something like: 

```The same considerations regarding authentication, integrity protection, and nonce checks described in this section and Section 7 for Map-Requests used to create subscription state, apply for Map-Requests used to remove subscription state.```

Reply:
Works for me. Thanks.